controllers/datamovement_controller.go

```diff
@@ -214,40 +254,154 @@ func (r *DataMovementReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	go func() {
 		var cmd *exec.Cmd
 
-		// TODO: UserId/GroupId
-
-		if isTestEnv() {
-			cmd = exec.CommandContext(ctxCancel, "sleep", "1")
+		// Allow the data movement command to be overridden via the configmapa
```

Comment:
"configmapa" = "config map". Also, suggest a comment line length of 100 or 120 columns. We all have 1080 monitors and plenty of real estate